src/utilities/savedFilters.ts

```diff
@@ -28,6 +28,11 @@ export function isSameFilter(filterA: SavedSearchFilter, filterB: SavedSearchFil
   return true
 }
 
+export function isEmptyFilter(filter: SavedSearchFilter): boolean {
+  const emptyFilters = { state: [], flow: [], deployment: [], workPool: [], tag: [] } satisfies SavedSearchFilter
+  return isSameFilter(filter, emptyFilters) && filter.startDate === undefined && filter.endDate === undefined
```

Comment:
I wonder why `isSameFilter` doesn't check the dates 🤔

Reply:
Yeah I updated that at first. my best guess is the way it's written atm leaves a convenient way for the `isSameFilter(oneWeekSavedSearch, emptyFilter) == true`. but since I'm changing that a little... it _might_ make sense to just extend the isSameFilter? 

🤔

Reply:
Could be a nice improvement. Your explanation sounds right.

Reply:
think I'm gonna leave this as-is for now so as to not break existing functionality. my brain is rolling over a little trying to figure out how to best fold startdate/enddate into `isSameFilter` w/o breaking the current behavior where the dates are ignored and either an empty filter _or_ the explicit filter match the current default view.